templates/CRM/Core/BillingBlock.js

```diff
@@ -9,14 +9,15 @@
    */
   function civicrm_billingblock_creditcard_helper() {
     $(function() {
-      $.each(CRM.config.creditCardTypes, function(key, val) {
-        var html = '<a href="#" title="' + val + '" class="crm-credit_card_type-icon-' + key + '"><span>' + val + '</span></a>';
+      $.each(CRM.config.creditCardTypes, function(card_type_key, val) {
+        card_type_css = card_type_key.toLowerCase();
```

Comment:
this line should not be required now - we have a list of names that are already lower case

Reply:
@eileenmcnaughton We've just tested against master and logging `CRM.config.creditCardTypes` return following JSON string.

`{Visa: "Visa", MasterCard: "MasterCard", Amex: "Amex", Discover: "Discover"}`

So the first character of each keys are still in uppercase.